src/view/screens/Settings/index.tsx

```diff
@@ -752,7 +754,7 @@ export function SettingsScreen({}: Props) {
           accessibilityRole="button"
           accessibilityLabel={_(msg`Export my data`)}
           accessibilityHint={_(
-            msg`Download Bluesky account data (repository)`,
+            msg`Opens modal for download Bluesky account data (repository)`,
```

Comment:
Similar here, I think "for downloading" would work better?
```suggestion
            msg`Opens modal for downloading Bluesky account data (repository)`,
```